Chummer/Backend/Equipment/WeaponAccessory.cs

```diff
@@ -186,6 +187,10 @@ public void Create(XmlNode objXmlAccessory, Tuple<string, string> strMount, int
             if (!objXmlAccessory.TryGetMultiLineStringFieldQuickly("altnotes", ref _strNotes))
                 objXmlAccessory.TryGetMultiLineStringFieldQuickly("notes", ref _strNotes);
 
+            String sNotesColor = ColorTranslator.ToHtml(ColorManager.HasNotesColor);
+            objXmlAccessory.TryGetStringFieldQuickly("notesColor", ref sNotesColor);
+            _colNotes = ColorTranslator.FromHtml(sNotesColor);
+
```

Comment:
Is this necessary in the `Create` method? Would there need to be data-definable colors for specific weapon accessories?